Project.toml

```diff
@@ -4,16 +4,21 @@ version = "0.9.0-dev"
 
 [deps]
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
-Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
+[weakdeps]
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+
+[extensions]
+FixedPointNumbersStatisticsExt = "Statistics"
+
 [compat]
-Requires = "1"
 julia = "1"
```

Comment:
If we follow the principle, `Statistics` should be in `[compat]`.
This is especially the case where even patch versions need to be included (e.g. `Statistics = "1 - 1.11.1"`).

Reply:
Fixed in https://github.com/JuliaMath/FixedPointNumbers.jl/pull/277/commits/6c2a68cc9513d092be75289e840daac58fc3c2e0.

Reply:
To my surprise, it seems that the hyphen notation is not supported for julia v1.0.
Sorry for misleading you.

Reply:
Yeah, `< 1.11.2` could be a solution (cf. [Pkg.jl v1.0 documentation](https://pkgdocs.julialang.org/v1.0/#Inequality-specifiers-1)). However, `"1 - 1.11.1"` looks better to me because it has a lower bound. Is there any chance to drop support for Julia 1.0?

Reply:
Since Statistics.jl is stdlib, the lower bound is practically meaningless.
I don't intend to continue actively supporting v1.0, but I don't think it has caused any critical problems yet.

Reply:
Also, this is very trivial, the uppercase "S" of Statistics precedes the lowercase "j" of julia.

Reply:
Thanks for the comments! I have updated the compat bound for Statistics.jl